### PR TITLE
[2.x] Fix CI 

### PR DESCRIPTION
**Pull request by dhmlau**
### Description

It includes increasing the mocha timeout, and uses the environment variable to get the db instance. 

#### Related issues
connect to https://github.com/strongloop/loopback-datasource-juggler/pull/1342

### CODE REVIEW

**Comment by dhmlau** [comment]
@slnode  test please

**Comment by dhmlau** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@slnode test please

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "./node_modules/.bin/mocha --timeout 5000 -R spec",
+    "test": "./node_modules/.bin/mocha --timeout 9999 -R spec",
```
Comment (ssh24) [inline package.json]: Do we know why the tests are failing on a smaller timeout? Extensively increasing the timeout to this big is not a good solution.

Comment (dhmlau) [inline package.json]: I was comparing the changes of the master branch to this branch, and the timeout there was 9999.  And it seems to be working here.

Comment (dhmlau) [inline package.json]: to be precise, i'm backporting the changes in `master` branch.  If `9999` is not ok for timeout, we need to fix for `master` branch too.

Comment (ssh24) [inline package.json]: Yeah :\ It should not have been `9999` in master to begin with.

Comment (dhmlau) [inline package.json]: @loay, any insights?

Comment (loay) [inline package.json]: Some tests take more time than another .. so if increasing the timeout fixes the issue, then I think we should increase it. If it is only one or two tests that need the extra timeout, then maybe we can define the extra timeout for those specific tests. If they are too many, then I am fine with increasing the timeout

**Comment by dhmlau** [comment]
@slnode  test please

**Comment by ssh24** [comment]
@slnode test please

**Pull request merged**